api/envoy/config/filter/listener/tls_inspector/v2/tls_inspector.proto

```diff
@@ -3,6 +3,7 @@ syntax = "proto3";
 package envoy.config.filter.listener.tls_inspector.v2;
 
 import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
```

Comment:
I don't see where we're using this import.  The proto_format script may print a warning.